File: fix_perms.py
with open('api/settings/get_permissions_groups.php', 'r', encoding='utf-8') as f:
    lines = f.readlines()
# Keep lines 1-217 (0-indexed: 0-216) and 558+ (557 onwards)
keep = lines[:217] + lines[557:]
with open('api/settings/get_permissions_groups.php', 'w', encoding='utf-8') as f:
    f.writelines(keep)

File: fix_perms.php
<?php
/**
 * EN: Handles application behavior in `fix_perms.php`.
 * AR: يدير سلوك جزء من التطبيق في `fix_perms.php`.
 */
$f = __DIR__ . '/api/settings/get_permissions_groups.php';
$lines = file($f);
$keep = array_merge(array_slice($lines, 0, 217), array_slice($lines, 405));
file_put_contents($f, implode('', $keep));
echo "Done.\n";
